dashboard/src/api/api.js

```diff
@@ -1,13 +1,22 @@
+export function IsUnauthorized(e) { 
```

Comment:
FYI in most JS styles only classes and components use leading uppercase. So this should be `isUnauthorized`.

Reply:
changed